spyderlib/config/main.py

```diff
@@ -39,7 +39,7 @@
     (_("Patch and diff files"), ('.patch', '.diff', '.rej')),
     (_("Batch files"), ('.bat', '.cmd')),
     (_("Text files"), ('.txt',)),
-    (_("reStructured Text files"), ('.txt', '.rst')),
+    (_("reStructuredText files"), ('.txt', '.rst')),

```

Comment:
Honestly, I don't have any opinion on that... the contributor (Benjamin) seems right to point out this minor typo. If it allows Spyder to be translated to a new language, I'm OK with it :)